core/server/config/index.js

```diff
@@ -1,11 +1,13 @@
 var Nconf = require('nconf'),
     path = require('path'),
-    debug = require('debug')('ghost:config'),
+    _debug = require('debug'),
```

Comment:
We could now use ignition.

`require('ghost-ignition').debug('config')`

But we could do it as separate task and remove the debug dependency from Ghost.